src/create_helpers.ts

```diff
@@ -166,6 +166,20 @@ function handleComment<T extends ts.Node>(doclet: IDocletBase, node: T): T
         const params = handleParamsComment(doclet);
         const returns = handleReturnsComment(doclet);
 
+        if ('isEnum' in doclet && doclet['isEnum'] === true)
+        {
+            if ('properties' in doclet)
+            {
+                const enumProperties = (doclet['properties'] as Array<IDocletBase>);
+                // @ts-ignore
+                const enumMembers = node['members'];
```

Comment:
Not sure if there's a safe way to call `members` on `ts.Node`. I tried to cast using `as` but it's not working.
Without the `@ts-ignore` directive, I get the following warning/error:

```
TS7017: Element implicitly has an 'any' type because type 'Node' has no index signature.
```